### DISCUSSION TITLE
run cmd/powershell command

### DISCUSSION THREAD

**Discussion by AndiLeni** [composer/composer#10627] [Q&A / Support]
Hello,

I wanted to ask whether it is possible to run multiple commands and keep them running from a custom script.
F.e. I would like a "sass" command to start two powershell windows which keep compiling my sass files.

````json
"scripts": {
    "sass": "start powershell {sass --watch styles.scss css/styles.css; Read-Host} && start powershell {sass --watch styles.scss css/styles.css; Read-Host}"
},
````
unfortunately, composer does not keep them alive but instead closes them directly after creation.
Is this somehow possible with composer, or should I change to npms "run script" or create a custom batch file?

**Comment by Seldaek** [comment]
IMO a custom script makes more sense for this, than using the dependency manager as a launcher for all the things.
